main/main_views/order_detail.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import generics
+from main.models import Order
+from main.serializers import OrderDetailSerializer
+
+
+# TODO: Change to Retrieve API View
+class OrderDetail(generics.RetrieveUpdateDestroyAPIView):
```

Comment:
You change it dude. I'm going to bed.
On Jun 10, 2015 10:31 PM, "Jared Mackey" <notifications@github.com> wrote:

> In main/main_views/order_detail.py
> <https://github.com/Pycart/PyCart-Backend/pull/43#discussion_r32189255>:
>
> > @@ -0,0 +1,12 @@
> > +from rest_framework import generics
> > +from main.models import Order
> > +from main.serializers import OrderDetailSerializer
> > +
> > +
> > +# TODO: Change to Retrieve API View
> > +class OrderDetail(generics.RetrieveUpdateDestroyAPIView):
>
> Please change to RetrieveAPIView as we don't want the ability to change
> and order from a non-admin type view. Also, please add authentication class
> and permission class to the view.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Pycart/PyCart-Backend/pull/43/files#r32189255>.
>